src/main/java/rx/swing/sources/WindowEventSource.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2015 Pedro Larroy
```

Comment:
Any reason why your name is used here? We've been writing Netflix in every other file.